package.json

```diff
@@ -40,7 +40,9 @@
     "release": "lerna publish",
     "test": "jest",
     "test:cov": "jest --coverage",
-    "test:ts": "lerna exec --concurrency=1 --no-bail -- 'pwd && tsc -p . --noEmit'"
+    "test:ts": "lerna exec --concurrency=1 --no-bail -- 'pwd && tsc -p . --noEmit'",
+    "link-packages": "lerna exec -- '[[ -d \"./dist\" ]] && (cd dist && yarn link) || yarn link'",
+    "unlink-packages": "lerna exec -- '[[ -d \"./dist\" ]] && (cd dist && yarn unlink) || yarn unlink'"
```

Comment:
I don't think these scripts are necessary. You only link the packages once, and it's not hard to unlink them manually. Also, the directory check is unnecessary since `yarn` triggers `yarn build` (which creates the `dist` folders).

Reply:
> the link/unlink package process was more difficult since some packages don't have dist folders

Sorry, I just noticed what you said. I think using `--no-bail` will ignore the lack of a `dist` folder in `packages/envinfo`. You could also use `--ignore "*/envinfo"`. I'll update the contributor guide.